Update README.md (#257)
DataDog/ansible-datadog: README.md

  sysprobe_socket: /opt/datadog-agent/run/sysprobe.sock
```

> Once modification completed, follow the steps below:

Reviewer: ```suggestion
Once modification completed, follow the steps below:

```

> Once modification completed, please start system-probe and restart the Datadog Agent follow the steps below:

Reviewer: ```suggestion
Once modification completed, follow the steps below:
```

> 1. Start the system-probe: `sudo service datadog-agent-sysprobe start` Note: If the service wrapper is not available on your system, run the following command instead: sudo initctl start datadog-agent-sysprobe

Reviewer: ```suggestion
1. Start the system-probe: `sudo service datadog-agent-sysprobe start` Note: If the service wrapper is not available on your system, run the following command instead: `sudo initctl start datadog-agent-sysprobe`
```

2. [Restart the Agent](https://docs.datadoghq.com/agent/guide/agent-commands/#restart-the-agent) with `sudo service datadog-agent restart`

3. Enable the system-probe to start on boot: `sudo service enable datadog-agent-sysprobe`

> You may also follow this [documentation](https://docs.datadoghq.com/network_performance_monitoring/installation/?tab=agent#setup) to setup NPM manually.

Reviewer: ```suggestion
You may also follow the [Datadog Network Performance Monitoring documentation (NPM)](https://docs.datadoghq.com/network_performance_monitoring/installation/?tab=agent#setup) to set it up manually.
```
